packages/backend/discovery/nova/ethereum/diffHistory.md

```diff
@@ -1,3 +1,35 @@
+Generated with discovered.json: 0xce1846ca8a71c542f62d345e25419cdb72e8f071
+
+# Diff at Fri, 08 Mar 2024 11:12:54 GMT:
+
+- author: torztomasz (<tomasz.torz@l2beat.com>)
+- comparing to: main@f09f798ebd2ae57f4c76e08114d608edf0a51c7b block: 19132347
+- current block number: 19390070
+
+## Description
+
+Provide description of changes. This section will be preserved.
```

Comment:
provide the description that the handler has been changed and not that the actual values have been changed, do the same for others

Reply:
It can not be added due to some bug, ask @mateuszradomski about it.